### PR TITLE
[2.0.2] zh, en: update dmctl help message

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)

align with https://github.com/pingcap/dm/pull/1437

**Waiting for DM 2.0.2 release**

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v3.0 changes)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by lance6716** [comment]
seems https://github.com/pingcap/docs-dm/blob/master/en/dmctl-introduction.md is not align with https://github.com/pingcap/docs-dm/blob/master/zh/dmctl-introduction.md . Will update it later

File: en/dmctl-introduction.md [hunk 1]
```diff
@@ -41,18 +41,18 @@ Usage:
 Available Commands:
   check-task      Checks the configuration file of the task.
   get-config      Gets the configuration.
-  handle-error    skip/replace/revert the current error event or a specific binlog position (binlog-pos) event.
-  help            Help about any command.
+  handle-error    `skip`/`replace`/`revert` the current error event or a specific binlog position (binlog-pos) event.
+  help            Help about any command
```
Comment (CharLotteiu) [inline en/dmctl-introduction.md:45]:
```suggestion
  help            Help about any command.
```

File: en/dmctl-introduction.md [hunk 2]
```diff
@@ -41,18 +41,18 @@ Usage:
...
   list-member     Lists member information.
   offline-member  Offlines member which has been closed.
-  operate-leader  evict/cancel-evict the leader.
-  operate-schema  get/set/remove the schema for an upstream table.
-  operate-source  create/update/stop/show upstream MySQL/MariaDB source.
-  pause-relay     Pauses DM-worker's relay unit.
+  operate-leader  `evict`/`cancel-evict` the leader.
+  operate-schema  `get`/`set`/`remove` the schema for an upstream table.
+  operate-source  `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source.
+  pause-relay     Pauses DM-worker's relay unit
```
Comment (CharLotteiu) [inline en/dmctl-introduction.md:51]:
```suggestion
  pause-relay     Pauses DM-worker's relay unit.
```

File: en/dmctl-introduction.md [hunk 3]
```diff
@@ -41,18 +41,18 @@ Usage:
...
   pause-task      Pauses a specified running task.
-  purge-relay     Purges relay log files of the DM-worker according to the specified filename.
+  purge-relay     Purges relay log files of the DM-worker according to the specified filename
```
Comment (CharLotteiu) [inline en/dmctl-introduction.md:53]:
```suggestion
  purge-relay     Purges relay log files of the DM-worker according to the specified filename.
```

File: en/dmctl-introduction.md [hunk 4]
```diff
@@ -41,18 +41,18 @@ Usage:
...
   query-status    Queries task status.
-  resume-relay    Resumes DM-worker's relay unit.
+  resume-relay    Resumes DM-worker's relay unit
```
Comment (CharLotteiu) [inline en/dmctl-introduction.md:55]:
```suggestion
  resume-relay    Resumes DM-worker's relay unit.
```

File: zh/dmctl-introduction.md [hunk 5]
```diff
@@ -48,11 +48,11 @@ Available Commands:
   operate-leader  `evict`/`cancel-evict` the leader.
   operate-schema  `get`/`set`/`remove` the schema for an upstream table.
   operate-source  `create`/`update`/`stop`/`show` upstream MySQL/MariaDB source.
-  pause-relay     pause DM-worker's relay unit
+  pause-relay     Pauses DM-worker's relay unit
```
Comment (CharLotteiu) [inline zh/dmctl-introduction.md:51]:
```suggestion
  pause-relay     Pauses DM-worker's relay unit.
```

File: zh/dmctl-introduction.md [hunk 6]
```diff
@@ -48,11 +48,11 @@ Available Commands:
...
   pause-task      Pauses a specified running task.
-  purge-relay     purge relay log files of the DM-worker according to the specified filename
+  purge-relay     Purges relay log files of the DM-worker according to the specified filename
```
Comment (CharLotteiu) [inline zh/dmctl-introduction.md:53]:
```suggestion
  purge-relay     Purges relay log files of the DM-worker according to the specified filename.
```

File: zh/dmctl-introduction.md [hunk 7]
```diff
@@ -48,11 +48,11 @@ Available Commands:
...
   query-status    Queries task status.
-  resume-relay    resume DM-worker's relay unit
+  resume-relay    Resumes DM-worker's relay unit
```
Comment (CharLotteiu) [inline zh/dmctl-introduction.md:55]:
```suggestion
  resume-relay    Resumes DM-worker's relay unit.
```

**Review by CharLotteiu — COMMENTED** [review]
A sentence should end with a period. Rest LGTM.

**Comment by lance6716** [comment]
I'll update the code and then commit the changes for helping message

File: en/dmctl-introduction.md (hunk 1 above)
Comment (lance6716) [reply en/dmctl-introduction.md:45]: This is a built-in message, could we change it later

Comment (CharLotteiu) [reply en/dmctl-introduction.md:45]: We'd better unify the text format. How about now?

Comment (lance6716) [reply en/dmctl-introduction.md:45]: waiting https://github.com/pingcap/dm/pull/1483

**Comment by lance6716** [comment]
/unlabel 
status/DNM

**Review by CharLotteiu — APPROVED** [review]

**Pull request merged**